src/System.Web.HttpUtility/src/System/Web/Util/HttpEncoder.cs

```diff
@@ -283,45 +283,14 @@ internal static byte[] UrlDecode(byte[] bytes, int offset, int count)
                 }
                 else if (b == '%' && i < count - 2)
                 {
-                    if (bytes[pos + 1] == 'u' && i < count - 5)
-                    {
-                        int h1 = HttpEncoderUtility.HexToInt((char)bytes[pos + 2]);
```

Comment:
.Net core also has same piece of code here: https://github.com/dotnet/corefx/blob/master/src/System.Web.HttpUtility/src/System/Web/Util/HttpEncoder.cs#L341 and that's not this PR's concern
Note the function signature difference, the function I modified is UrlDecode(byte[] bytes, int offset, int count), not UrlDecode(byte[] bytes, int offset, int count, Encoding encoding)